src/web/modules/transactions/update/update.http.ts

```diff
@@ -145,6 +150,36 @@ const validateAndAddDefaults = async function validateAndAddDefaults(csv: string
         if (row.transaction_type === 'refund' && !row.parent_transaction_id) {
           return cb(null, false, 'parent_transaction_id is required when transaction_type is \'refund\'')
         }
+        if (row.captured_date && !moment(row.captured_date, moment.ISO_8601).isValid()) {
+          return cb(null, false, 'captured_date is not a valid ISO_8601 string')
+        }
+
+        if (row.refund_amount_refunded && !parseInt(row.refund_amount_refunded)){
+          return cb(null, false, 'refund_amount_refunded must be a number')
+        }
+
+        if (row.refund_amount_available && !parseInt(row.refund_amount_available)){
+          return cb(null, false, 'refund_amount_available must be a number')
+        }
+
+        if (row.event_name === 'PAYMENT_STATUS_CORRECTED_TO_SUCCESS_BY_ADMIN') {
+          if (!row.captured_date){
+            return cb(null, false, `captured_date is required when event_type is '${row.event_name}'`)
+          }
+          
+          if (!row.refund_status){
+            return cb(null, false, `refund_status is required when event_type is '${row.event_name}'`)
```

Comment:
Sorry missed in first round, can we refer to `event_name` to tie with csv field name (and for usages below)
```suggestion
            return cb(null, false, `refund_status is required when event_name is '${row.event_name}'`)
```

Reply:
Good spot - done